Robinhood/Robinhood.py

```diff
@@ -161,10 +161,19 @@ def logout(self):
             warnings.warn('Failed to log out ' + repr(err_msg))
 
         self.headers['Authorization'] = None
-        self.auth_token = None
+        self._auth_token = None
 
         return req
 
+    @property
+    def token(self):
+        return self._auth_token
+
+    @token.setter
+    def token(self, token):
+        self._auth_token = token
+        self.headers['Authorization'] = 'Token ' + self._auth_token
```

Comment:
changing the token behavior should probably come with a test.

I like the change but would like to validate that setter is working as expected for users

Reply:
I didn't realize there were tests at this point. I first made this as a PR to a fork that I was using, then when seeing his PR didn't get merged directly, I submitted it here.

Tests would be ideal. I haven't written any, just determined it worked for me.